components/dashboard/Carousel/CarouselWrapper.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import "./carouselWrapper.scss";
+
+const responsive = {
+	desktop: {
+		breakpoint: { max: 5000, min: 2000 },
+		items: 12,
+	},
+	iPadPro1024px: {
+		breakpoint: { max: 1199, min: 1024 },
+		items: 4,
+	},
+	iPadMini768px: {
+		breakpoint: { max: 1023, min: 768 },
+		items: 4,
+	},
+	mobile414px: {
+		breakpoint: { max: 767, min: 375 },
+		items: 3,
+	},
+	mobile: {
```

Comment:
dość dziwne nazewnictwo zmiennych, co jeśli ktoś nie używa iPada? :D 
możesz nazwać po prostu tablet i smallDesktop
a mobilka powinna iść od 320, nie ma nic mniejszego

Reply:
Miałam zajęcia z UX na podyplomówce i gość na podstawie statystyk polecał robić od Galaxy 280px, wydało mi się to sensowne, dlatego też tak robię. W praktyce tak się jednak nie robi?

Reply:
yyy, obecnie się nawet mówi, że najmnijeszy jest 375px 😅 ale zawsze biorę pod uwagę tego iphone 5 (choć stary),

WCAG:
```
Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions for: 

Vertical scrolling content at a width equivalent to 320 CSS pixels;
``` 

i całkiem śmieszny thread: https://www.reddit.com/r/webdev/comments/107vp0c/so_just_to_clarify_were_all_collectively_ignoring/ XD